vda5050_msgs/vda5050_msgs_mapping_rules.yaml

```diff
@@ -0,0 +1,27 @@
+-
+  ros1_package_name: 'vda5050_msgs'
```

Comment:
these are incomplete though aren't they? 
if we want to add those we probably would want it for all the messages types in here.
The advantage of https://github.com/ipa320/vda5050_msgs/tree/ros2/vda5050_serializer/vda5050_serializer is that it's not hardcoded and therefore actually more resistent to change in the message definitions here. I'm not sure if something like that is supported in ros1 ros2 bridge, otherwise I probably would keep these mapping files in your specific repo or at least autogenerate them in ci in here.

Reply:
I just added those message types that I wanted the ros1_bridge to be able to bridge for a specific customer. This is the way that Kemal suggested me to do. I have no clue how the ros1_bridge could use our serializer to automatically translate dromedarCase messages to snake_case messages. IMO it is necessary to either add the mapping rules to
- our ros1_bridge repo (which might not be the nicest thing)
- or here
AFAIK it's rather complicated to add the mapping rules directly to our customer repo given the fact that we cannot easily add the mapping rules simply as an argument but need to rebuild the ros1_bridge docker image image with the mapping rules.